sds-aks-release.drawio

```diff
@@ -0,0 +1 @@
EnTv+/KHOlYfC5t2kBYN+YJi3tGLuafUxYP+aWOh7Ees+nPuvxPjtJAaL0B/wG87D8Z8rJe49Hg+kxP+QdGBY9o5GDHpHI4r6gcIB++e4WB+YH3+N83+8Vfp4er82pI9+9xzt/8r1ipJv3SDoM3++a8Qd/6m+jr/Lgx85Df02TvuLXJMv27/vYdw+5kn0K5kX++7M+/dMpHsbSdNhVSiibx9y2q+RxfGyDL6Pu5y+sRmescDf9ZYTb0Mnf6IY/fYa38Vf/nh+0IfE+4d7yugvyYk3IuGTw+u1VPi8y+sr/GU/S3D8IloPY95yNU6/CQh+cTzKEjcM/x55affZC0LktV0dgZNc7RXBfWru/7jzDkOwt3R9MX9+lvMO/bf49dcofn0pNn1mDJa+44v3qn19zBe/THL1rxft+6zi+DI0xX4paIrf+95+GqHpr6X0B4J+a9v88y2bB+zzK2dE4ezbUs8bwPHF8fiz2+mvxpM3sP5m/DuZ/b9OGvdnF8Lf49xXy4j824voZ0nRl3jTe5eKMtRbwEa81Ip+7xpO6sbNhXxhfVDk3xtPfD4jkWKIvzeeRT87nr5xO9+M/9vr+3HKw6PA01OuI8CKxZuVT527En4RPK2gGWSlg///gPSFIZ4/rrW9L3/9/0/DXzImbytPn+4CnvrpRk+DHkqbd3b9/FWey/1C/qyI/C4uIQy5TXX842+5hN7f4/Min/6tRP2ZdRw4eiNsyAcxqx9ZiUr8cxyBn5Q/jHl/l2LUXwhGvxhXPws/fHthwY1aZj6vNm/HY/jnYTeN4p8b/z4w+oUI/xa5/ZQiN+IXE44/t4cO8i3C8WFA//vYWj/VOfXVQX+afhexeG9WvTguX5j0pTD2O5tVNy1LMPrz9f03w5kv2DwY9Znh7yRb7wGo3l1/to3O3SMp+8tGXI6d/5+gDKMXMyWqF330ZK18ewTwGzJZ8beeKIz4yfnT+H3m+xetUqiZ/nzKGoOWKYpWw70N+raa7872/FTk99/e4Ps85rd30/s6Y/r3TdvD2bfSE2Pv8yrRRz10X5z935+HyQc8fEOPf1N6/+kpvcCa+IDeaFQE+cr6k3dL6n3Qv/lR7n9V//ly+umK/2vJvhTL3lHvgW2AYsQ98d4t2/dB11V+qRmLP36d1Lhvmv1vdoOiNz5QgCqB6MI/foi/nvi/Soh7+4v3c4a+WIyvaPfibnhNsydD/kXQG28F/Wsq3roDbruQxBEVBH8p9O5UQ0iz/n/hY/oGcUi/zdnAn2Ogr1fTSzLim+4k9F/T9O8VQd0nZD1wTP629EBv+gTjD5D9j6XHvSPsHhb9xvRg366Pj+b1T6PHvaWFf2MA0HscAfzjdSzwaU6Rq/ETe3mSjU8/z8uifMK5b4Z8so+Qx0YWeZ1eCAhXwFzO4InF9WxdgpEfj17mgLzOAjgzh3/DByPhW5Nwp5YvjEU/jn1hh//qMtinyzxN9aMbPM30x29elgE48XohwO+vSwGevy4GeAa9Hj4vCHjiaUnAk58WxculXxYG/Pbt0oAj4OKA37xeHh9/+bxE4PevXulpAXzNpH386uN8fVolJFwnH0f+CTtbvVxmfHWeoj+df1o3n6b51VfP6+fhd2AdfTx/eHX/W2peDz+S9PXJt4z2PO6OI78yxv3byjucpW7k3X3Hlh8r7+7B9YMMp9+XHtTbghMUvbd2fiw97r0kD2Ijvy09qNvIFPOz8fK95+BB75vflh63sWSMeODK+bEEuW/h/yAP/bclCPlWXpEU9tMJcp8x96A8+7clCH4TpiRo5ifT4z78x/4v0YN+q0HoBw0lfig96AcusAe1478tQQgGe0sQ5CdDXvqBC+x/yQd2m5PwqNnPjyXIvQ/sJQjxv0EQ+m2iO/myKdRPI8gDhfHL56qyf7xJn8L+eKdE1Z/UyfGzuVj/lFxW+qaJ9Reyp26HvyjOd60AebA58ztXgFhFUgAug4/2u9SAvEiQ71IDQpDMM5//7V0i32L1d4mDPmSqR/lbN0SFOcbV31Qx3zy75Eu6z8deoeiHe3hGEh8wBCdxloGijGEeRP9vh7z4Ob976+FHq/O7IYWvRAQ+QxIkco8sBIECn4er5LNc8WWk8DliPOi89ldjvn8j6C9T4yVhqcuzl41LvkiYpxnfPb80/gDDMUEUBI9oI1w/70GDjyHkDxT2qskpe1OegX6gMeTT5wWkve7SgH3sqPcm8474gH2HHJqHb/nL7FH7Xq0a/oG9uT8n3r5Y4/+yD8Mv0hD+l+kH/9X7LH6n0rbfmO9+4V0cqZcrvsAW+vOl7bfj6eeeRn81nr1J2/rW8d97z8fHuvdBq9p/DZW/NlQ+J7i+x972CH272c+vVKv+mIXuPW93NPwpdgmF3vgo7z1ijzoOU99Bv33W0fI+ZscNuI3QkIzorzdHWIrGvW8wOz5S/Rta+99N/0vi5Jv8ffK9pv8+xriLfHBCKsv0jhC/Yn7+3yXES3TxPpj1Xp23Hz/xr+rIQG8RwUvJxw+QGJ/dO+f3kBjfbiX/OInx+Inv86h+C4nx9YT48RIDsbZJNDXrhqEY1or/I1LE8mu2/nklML5aEHytbPlLBmWYt2CNeMCxAPh/wJlPvhz0wby9RLO//w4oX+HpfGtCf0GIPJYNN5Ik9CImfuhYowIm8uOHrPpZwn+Tc/Npxn/oNN+7MJ86kSAz1bib8DtB/Gp+Hyz8WyHy0gzktmvIo+Yg1yJ4+HKwz6yXFFENe4SoH8s9/2NEdZ8E0V9vMvk3yfJi275ZJvfezEdFxO/WKOQll+TfIuJfs4j4+7AcAHO3e87c890PLSBmHuDef+i2QN+JRCh+ty3Q19no74Y0vqK/wE9CGjc1H4/yrX4Q0ngIKr/Ct/ErAw3yLwj684DGw1m+d2HoQHb8tjjj66nyk3HGQ2I9cq3/CzN+bZjxzRz3U2HGY/P+gc33O8GMbyfRj4QZfzxXsb4KsXyqX8UX/wc=</diagram></mxfile>
```

Comment:
I'm not sure if `{{product}}` best describes this:
![image](https://user-images.githubusercontent.com/21194782/136523765-61f1d339-f174-406d-864d-69650f31e554.png)

it's really `$product-$component` which might be a bit name, `$app`, or `$app-name` might better

This isn't handling the different cluster names / environments between SDS and CFT at all which will create confusion, e.g:

```
dev = preview
test = perftest
stg = aat
```

Step 3 is wrong:
![image](https://user-images.githubusercontent.com/21194782/136524215-cc75be3f-971a-41ea-999a-ef9ec7c8e3cf.png)

it should be a PR image not a staging image

I think the diagram needs at least an intro as well not just dropping it on the page but open to suggestions

Reply:
this includes pr number as well:
example `hmctspublic.azurecr.io/cmc/citizen-frontend:pr-3265-8cdc94e-20211005100511`

![image](https://user-images.githubusercontent.com/21194782/136531432-6962f5c5-3b0e-4556-b301-c36267e7f0cd.png)

Reply:
@njrich28 See section from here https://github.com/hmcts/hmcts.github.io/pull/129/files#diff-d429c650c892d8f9c7468cc618791c1cd630e7722bd857d82f0cd8554c53dbe1R51 

Does that make sense / reflect reality?